rules/eslint/es6/airbnb.js

```diff
@@ -35,18 +35,24 @@ module.exports = {
     "no-class-assign": 0,
     // disallow modifying variables that are declared using const
     "no-const-assign": 0,
+    // disallow duplicate name in class members
+    "no-dupe-class-members": 0,
     // disallow to use this/super before super() calling in constructors.
     "no-this-before-super": 0,
     // require let or const instead of var
     "no-var": 2,
     // require method and property shorthand syntax for object literals
     "object-shorthand": 0,
+    // suggest using arrow functions as callbacks
+    "prefer-arrow-callback": 0,
```

Comment:
Can we make this one a warning?  There are definitely times when the context of a callback matters.